### ISSUE TITLE
Ugly headings in POD

### ISSUE THREAD

**Issue by van-de-bugger** [rjbs/Test-Deep#38]
`Test::Deep` POD uses lengthy headings like these:

```
=head3 $ok = cmp_deeply($got, $expected, $name)
=head3 $ok = cmp_bag(\@got, \@bag, $name)
=head3 $ok = cmp_set(\@got, \@set, $name)
```

It provides useful information about function arguments, but it makes links from other PODs terrible:

```
see L<Test::Deep/"$ok = cmp_deeply($got, $expected, $name)">.
```

It would be nice if `Test::Deep` POD headings do not include result and parameters, e. g.:

```
=head3 cmp_deeply
    $ok = cmp_deeply($got, $expected, $name)
...
=head3 cmp_bag
    $ok = cmp_bag(\@got, \@bag, $name)
...
=head3 cmp_set
    $ok = cmp_set(\@got, \@set, $name)
...
```

Such headings are common for other modules (see [Sort::ByExample](https://metacpan.org/pod/Sort::ByExample) or [String::Flogger](https://metacpan.org/pod/String::Flogger) to name a few).
In such a case links from other PODs will be much simpler:

```
see L<Test::Deep/"cmp_deeply">.
```

**Comment by kspice** [comment]
This issue was fixed fd2fb8055b223a4c38fba1579a144a7f9a5f7d0e - investigated as part of the CPAN Pull Request challenge team [CV-Library](https://github.com/cv-library)

**Comment by rjbs** [comment]
…and so closed!
